packages/console/src/pages/SignInExperience/PageContent/utils/parser.ts

```diff
@@ -122,6 +122,7 @@ export const sieFormDataParser = {
       mfa,
       captchaPolicy,
       sentinelPolicy,
+      emailBlocklistPolicy,
```

Comment:
Do we still have these data fields in the `SignInExperienceForm`? I thought they were moved to "Security" pages